modules/ting/src/Search/TingSearchSort.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * @file
+ * The TingSearchSort class.
+ */
+
+namespace Ting\Search;
+
+/**
+ * Class TingSearchSort
+ *
+ * Represents a provider-independent sort-order for a query.
+ *
+ * @package Ting\Search
+ */
+class TingSearchSort {
+  protected $direction;
+  protected $field;
+
+  // TODO: Could be an enum.
+  const DIRECTION_ASCENDING = 'asc';
+  const DIRECTION_DESCENDING = 'desc';
+  // "NONE" should be used in situations where the it doesn't make sense to
+  // specify a direction for a sort. This can be necessary in cases where the
+  // provider simply does not support a direction for the sort, or where it
+  // does not make sense ie. a random sort.
```

Comment:
Note "random sort" have been removed.

Reply:
I know. Here is it just mentioned as an example.